merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -136,7 +136,16 @@ public List<BulkArgumentValidationResponse> validateActivityArgumentsBulk(
       final List<ActivityDirectiveForValidation> activities
   ) throws NoSuchMissionModelException, MissionModelLoadException {
     // load mission model once for all activities
-    final var modelType = this.loadMissionModelType(modelId.toString());
+    ModelType<?, ?> modelType;
+    try {
+      modelType = this.loadMissionModelType(modelId.toString());
+      // try and catch NoSuchMissionModel here, so we can serialize it out to each activity validation
+      // rather than catching it at a higher level in the workerLoop itself
+    } catch (NoSuchMissionModelException e) {
```

Comment:
The new deeper `catch` that can handle and map this exception to actual failed validations (which resolves them from `pending`, instead of just printing the error and restarting the loop.